app/models/plant_accession.rb

```diff
@@ -1,8 +1,10 @@
 class PlantAccession < ActiveRecord::Base
-  belongs_to :plant_line
+  belongs_to :plant_line, counter_cache: true
   belongs_to :plant_variety
   belongs_to :user
 
+  # TODO: add callbacks to update PlantVariety#plant_variety_accessions_count
```

Comment:
I removed this TODO. Since current implementation of `PlantVariety.table_data` does not seem to be slow I don't think it is that important to have this count cached at this point.